docs/advanced/backup_restore.md

```diff
@@ -0,0 +1,233 @@
+              objectStorage:
+                bucket: my-acm-backup-restore       <-- S3 bucket name
+                prefix: my-acm-backup               <-- S3 folder name to store the backup data
+              default: true
+              provider: aws
+```
+
+6. Create the following `BackupSchedule` resource to start backing up and store them in the S3 bucket every hour.
+
+```yaml
+      apiVersion: cluster.open-cluster-management.io/v1beta1
+      kind: BackupSchedule
+      metadata:
+        name: schedule-acm
+        namespace: open-cluster-management-backup
+      spec:
+        veleroSchedule: 0 */1 * * *         <-- every hour (cron scheduling)
+        veleroTtl: 12h                      <-- backup data time to live
+        useManagedServiceAccount: true      <-- this enables auto-recovery auto-reconnection of all management and hypershift clusters on service cluster fail-over
+```
+
+## Specifying resources to be backed up
+
+See https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/backup_and_restore/backup-intro#resources-that-are-backed-up for the list of ACM resources that are automatically backed up. If you want ACM backup to back up resources other than the ones listed in this document, you need to add the special label to the resources. In this example, I want to backup:
+
+- the manifestwork resources for creating HostedCluster and NodePool CRs on management clusters
+- the manifestwork resource for configuring htpasswd of the hosted clusters
+- any secrets or configmaps that are referenced by the HostedCluster and NodePool CRs
+
+Add the following label to all these manifestwork resources. The existence of this label without any value indicates to the ACM backup that this resource needs to be part of the backup. This label can always be added to the resources that the cluster service component creates to provision a new hypershift cluster. If ACM backup is not enabled, this label will just be ignored.
+
+```
+    cluster.open-cluster-management.io/backup: ""
+```
+
+
+## Configuring ACM on passive service cluster
+
+It is recommended that the passive cluster has identical set of operators and other configurations as the active cluster.
+
+1. Enable the backup(restore) feature in ACM.
+
+```
+      $ oc edit mch multiclusterhub -n open-cluster-management
+
+          - enabled: true
+            name: cluster-backup
+```
+
+2. Enable managed cluster auto-import recovery feature in MCE.
+
+```
+      $ oc edit mce multiclusterengine
+
+          - enabled: true
+            name: managedserviceaccount-preview
+```
+
+3. Create the restore cloud secret. This secret is used by the ACM restore to get access to the AWS S3 bucket.
+
+```
+      $ oc create secret generic cloud-credentials --namespace open-cluster-management-backup --from-file cloud=/Users/rokej/.aws/credentials
+```
+
+4. Create the following `DataProtectionApplication` instance.
```

Comment:
Create the [DataProtectionApplication resource](https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-dataprotectionapplication-custom-resource) and connect to the same storage location where the active service cluster had backed up the data.